hivemind/client/allreduce.py

```diff
@@ -150,9 +150,7 @@ async def accumulate(self, source: Endpoint, part: torch.Tensor) -> torch.Tensor
         return await self.averaged_part
 
     def _get(self, peer: Endpoint) -> averaging_pb2_grpc.DecentralizedAveragingStub:
```

Comment:
I believe _get is slightly overloaded here, how about _get_peer_stub?

Reply:
its mostly a convention with DHTProtocol._get, should we rename both?